object_manager_mx_record.go

```diff
@@ -41,47 +45,59 @@ func (objMgr *ObjectManager) GetMXRecordByRef(ref string) (*RecordMX, error) {
 	return recordMX, err
 }
 
-func (objMgr *ObjectManager) GetMXRecord(dnsview string, fqdn string) (*RecordMX, error) {
-	if dnsview == "" || fqdn == "" {
+func (objMgr *ObjectManager) GetMXRecord(dnsView string, fqdn string) (*RecordMX, error) {
+	if dnsView == "" || fqdn == "" {
 		return nil, fmt.Errorf("DNS view and fqdn are required to retrieve a unique mx record")
 	}
 	var res []RecordMX
 
 	recordMX := NewRecordMX(RecordMX{})
 
 	sf := map[string]string{
-		"view": dnsview,
+		"view": dnsView,
 		"name": fqdn,
 	}
 	queryParams := NewQueryParams(false, sf)
 	err := objMgr.connector.GetObject(recordMX, "", queryParams, &res)
 
 	if err != nil {
 		return nil, err
-	} else if res == nil || len(res) == 0 {
```

Comment:
There was a misunderstanding between us. I did mean that 'else' keyword is redundant here, but the 'if' clause is still necessary. Please, return this back, but without 'else' keyword.